src/folsom_cowboy_app.erl

```diff
@@ -33,19 +33,24 @@
 -define(APP, folsom_cowboy).
 
 start(_Type, _Args) ->
-    Dispatch = cowboy_router:compile(env(dispatch)),
+    {ok, _} = application:ensure_all_started(cowboy),
+
+    Dispatch = cowboy_router:compile(folsom_cowboy:dispatch()),
 
     Env = {env, [{dispatch, Dispatch}] },
 
-    ProtoOpts = case application:get_env(?APP, enable_jsonp, false) of
-        true ->
-            [Env, {onresponse, fun maybe_add_padding/4}];
-        false ->
-            [Env]
-    end,
+    ProtoOpts =
+        case application:get_env(?APP, enable_jsonp, false) of
```

Comment:
That's a good point. I'm just vary supporting a feature I don't see myself using.. Anyway, this is just a thought.